### PR TITLE
toast-and-grid

### PR DESCRIPTION
**Pull request by amantulsyan35**

### CODE REVIEW

File: css/components/toast.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
+.toast-container {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 99999;
```
Comment (Naman-Saxena1) [inline css/components/toast.css]: I think this is an overkill.

File: css/components/grid.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+.grid-3-column-layout {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  border: 1px solid rgb(0, 0, 0, 0.1);
+  padding: 1rem;
+  gap: 1rem;
+  width: 100%;
+  max-width: 56.5rem;
+  margin: 0 auto;
```
Comment (Naman-Saxena1) [inline css/components/grid.css:32]: You can add justify-items : center for better alignment.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

**Review by thenorthkun — APPROVED** [review]
Good code, merge ready👍

File: css/components/grid.css (hunk 2 above)
Comment (amantulsyan35) [reply css/components/grid.css:32]: have added a separate class for that

**Pull request merged**